### PR TITLE
added guide for mercurial

### PR DESCRIPTION
**Pull request by sourabh1031**
NOTE:
- Please fill out the template below 

## Did you read the instructions in the [README](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/README.md)?

> Y

## Have you read the instructions in [CONTRIBUTING](https://github.com/the-ethan-hunt/first-timers-guide/blob/master/CONTRIBUTING.md)?

> Y

## Have you written this article by yourself? Have you taken inspiration from anywhere and then written it in your words?

> [I have taken reference from [wikipedia](https://en.wikipedia.org/wiki/Mercurial) and [official guide](https://www.mercurial-scm.org/guide)]

### CODE REVIEW

**Comment by sourabh1031** [comment]
@the-ethan-hunt please review it

**Comment by sourabh1031** [comment]
@RN0311 i have updated the changes please review it

**Comment by RN0311** [comment]
@sourabh1031 : Now, it's better! Thanks for updating!
@the-ethan-hunt : Reviewed.

File: guides/Version_Control/Mercurial.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Mercurial
+Mercurial is one of the distributed version control system available today.If you are not aware of what a distributed version control is refer to [this](https://en.wikipedia.org/wiki/Distributed_version_control). The most commonly used Micrsoft Windows uses it. It is also used in developing various other linux distributions including FreeBSD and even macOS.
+
+## Installation
+Mercurial is available for all platforms. To download it visit their official download [page](https://www.mercurial-scm.org/downloads). It provides all the necessary information and guide for download and installion.
+
+##  Where shall i learn mercurial?
```
Comment (the-ethan-hunt) [inline guides/Version_Control/Mercurial.md]: *I

File: guides/Version_Control/Mercurial.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+The best resource to learn is the official [guide](https://www.mercurial-scm.org/guide). It will take you as a noob and will make you pro.
+
+## Features
+* design goals include high performance and scalability,
```
Comment (the-ethan-hunt) [inline guides/Version_Control/Mercurial.md]: Proper indentation is required here

File: guides/Version_Control/Mercurial.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+*  robust handling of both plain text and binary files
+*   advanced branching and merging capabilities
+*   easy transition from earlier version control e.g SVN
+
+## More stuff to grasp
+Mercurial is available both in command line and as well graphical user interface.For GUI you need to download various extensions.[TortoiseHg](https://en.wikipedia.org/wiki/TortoiseHg) is one of the popularly used extensions while there are many others IDE as well.
+
+## Want more resources ??
+* Git vs Mercurial - https://www.atlassian.com/blog/git/git-vs-mercurial-why-git
```
Comment (the-ethan-hunt) [inline guides/Version_Control/Mercurial.md]: Compare the pros and cons of using Git over Mercurial

**Comment by sourabh1031** [comment]
@the-ethan-hunt please review i have updated the guide

**Comment by the-ethan-hunt** [comment]
Merging

**Pull request merged**